modal/parallel_map.py

```diff
@@ -134,13 +134,21 @@ async def pump_inputs():
             logger.debug(
                 f"Pushing {len(items)} inputs to server. Num queued inputs awaiting push is {input_queue.qsize()}."
             )
-            resp = await retry_transient_errors(
-                client.stub.FunctionPutInputs,
-                request,
-                max_retries=None,
-                max_delay=10,
-                additional_status_codes=[Status.RESOURCE_EXHAUSTED],
-            )
+            while True:
+                try:
+                    resp = await retry_transient_errors(
+                        client.stub.FunctionPutInputs,
+                        request,
+                        max_retries=1,
+                        max_delay=101,
```

Comment:
I think maybe a higher `max_retries` maybe `7` could make sense -- then the user would be waiting for ~30 seconds between log lines which isn't too spammy.